docs/content/modeling/migrating/migrating-relations.mdx

```diff
@@ -193,11 +193,9 @@ Run a check in the API for Bethany to ensure correct access.
 
 Next, migrate the existing relationship tuples. The new relation makes this definition obsolete.
 
-Use the `read` API to lookup all relationship tuples with the `editor` name in your model.
+Use the `read` API to lookup all relationship tuples.
```

Comment:
Should we add a note in the step after that tuples not matching object type == document and relation == editor should be filtered out?